internal/server/recon/recognize_test.go

```diff
@@ -380,15 +379,15 @@ func TestRankAndTrimCandidates(t *testing.T) {
 					{
 						Tokens: []string{"Mountain", "View"},
 						Places: []*pb.RecogPlace{
-							{Dcid: "geoId/MTV2", Population: 102},
-							{Dcid: "geoId/MTV1", Population: 101},
+							{Dcid: "geoId/MTV2", Population: 9102},
+							{Dcid: "geoId/MTV1", Population: 9101},
 						},
 					},
 				},
 			},
 		},
 	} {
-		got := pr.rankAndTrimCandidates(c.tokenSpans)
+		got := pr.rankAndTrimCandidates(c.tokenSpans, 2)
```

Comment:
should we have an e2e test for Franklin County?

Reply:
SG - could do in a follow up CL.